### ISSUE TITLE
Learning rate scheduler of the main model

### ISSUE THREAD

**Issue by ZigeW** [cxcscmu/MATES#2]
Hi,

I'm confused about the learning rate scheduler used in training the main model. Is it the WSD scheduler mentioned in the paper?  Does the learning rate scheduler applied to each stage or the entire pretraining process?

Thanks

**Comment by yuzc19** [comment]
Hi! We use the WSD scheduler in the paper. 
https://github.com/cxcscmu/MATES/blob/e1c4fe1bdf7a73b833fb1cc092e3ea282917fa58/src/pretrain/pretrain.py#L319
It applies to the entire pretraining, so we totally have 2000 steps for warmup, 48000 steps for stable, and 200 steps for decay. But let's say if you want to evaluate 10k step performance (which is in the stable stage), it is *recommended* to run an additional decay stage since it will give you better performance.
